ledger/simulation/simulator.go

```diff
@@ -177,22 +175,22 @@ func (s Simulator) evaluate(hdr bookkeeping.BlockHeader, stxns []transactions.Si
 	return vb, nil
 }
 
-func (s Simulator) simulateWithTracer(txgroup []transactions.SignedTxn, tracer logic.EvalTracer) (*ledgercore.ValidatedBlock, []int, error) {
+func (s Simulator) simulateWithTracer(txgroup []transactions.SignedTxn, tracer logic.EvalTracer, signaturesOptional bool) (*ledgercore.ValidatedBlock, error) {
```

Comment:
minor: also related with `signaturesOptional` argument in function signature above.